src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/CachingHTTPClientMock.swift

```diff
@@ -42,6 +42,22 @@ final class CachingHTTPClientMock: CachingHTTPClient {
 		return config
 	}()
 
+	static let staticAppConfigMetadata: AppConfigMetadata = {
+		let bundle = Bundle(for: CachingHTTPClientMock.self)
+		// there is a test for this (`testStaticAppConfiguration`), let's keep it short.
+		guard
+			let fixtureUrl = bundle.url(forResource: "de-config-int-2020-09-25", withExtension: nil),
```

Comment:
Just FYI, the app configuration format will change in the next two sprints which will likely break the deserialization. However, with the default app configuration on the horizon we might have this covered!

Unless we have a need for a bespoke config, we could use the static default configuration, right?

Reply:
Good point. Made use if CachingHTTPClientMock.staticAppConfig